apps/condo/domains/billing/schema/BillingIntegrationOrganizationContext.js

```diff
@@ -85,7 +85,7 @@ const BillingIntegrationOrganizationContext = new GQLListSchema('BillingIntegrat
     hooks: {
         validateInput: async ({ operation, resolvedData, addValidationError }) => {
             // should have only one explicit (hidden = false) context in organization!
-            if (operation === 'create' && resolvedData['status'] === CONTEXT_FINISHED_STATUS) {
```

Comment:
Not the right way to do it!

Client must be allowed to request connecting another integration...

Reply:
Case is simple:
1. Client is using billing A (it has `Finished`) context
2. Client want to connect billing B instead (see https://confluence.in.doma.ai/pages/viewpage.action?pageId=1214678029)
3. We're creating second context with "InProgress" status, which allows marketing to get a new deal in AMO via meta base integration

Reply:
So, instead of restricting creating second context we should fix filtering in `useObject` by adding status filter:
https://github.com/open-condo-software/condo/blob/master/apps/condo/pages/billing/index.tsx#L29

Reply:
Why? Right now we do not support two billing integrations. We only support stragne UPS case, but its a HIDDEN billing + CONNECTED billing